2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_setup.py:_flush():71] setting env: {}
2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_init.py:_log_setup():371] Logging user logs to /home/aiotlab/projects/cuongdm/MEC1/wandb/run-20220313_001923-12q5qdv9/logs/debug.log
2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_init.py:_log_setup():372] Logging internal logs to /home/aiotlab/projects/cuongdm/MEC1/wandb/run-20220313_001923-12q5qdv9/logs/debug-internal.log
2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_init.py:init():404] calling init triggers
2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-03-13 00:19:23,140 INFO    MainThread:17669 [wandb_init.py:init():460] starting backend
2022-03-13 00:19:23,140 INFO    MainThread:17669 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-13 00:19:23,144 INFO    MainThread:17669 [backend.py:ensure_launched():216] starting backend process...
2022-03-13 00:19:23,151 INFO    MainThread:17669 [backend.py:ensure_launched():221] started backend process with pid: 18245
2022-03-13 00:19:23,153 INFO    MainThread:17669 [wandb_init.py:init():469] backend started and connected
2022-03-13 00:19:23,157 INFO    MainThread:17669 [wandb_init.py:init():533] updated telemetry
2022-03-13 00:19:23,176 INFO    MainThread:17669 [wandb_init.py:init():563] communicating current version
2022-03-13 00:19:24,406 INFO    MainThread:17669 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-13 00:19:24,407 INFO    MainThread:17669 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-13 00:19:24,708 INFO    MainThread:17669 [wandb_init.py:init():606] starting run threads in backend
2022-03-13 00:19:29,721 INFO    MainThread:17669 [wandb_run.py:_console_start():1810] atexit reg
2022-03-13 00:19:29,721 INFO    MainThread:17669 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-13 00:19:29,722 INFO    MainThread:17669 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-13 00:19:29,726 INFO    MainThread:17669 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-13 00:19:29,726 INFO    MainThread:17669 [wandb_init.py:init():633] run started, returning control to user process
2022-03-13 00:19:46,287 INFO    MainThread:17669 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-13 00:19:46,290 INFO    MainThread:17669 [wandb_run.py:_restore():1752] restore
2022-03-13 00:19:49,257 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:19:49,553 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:19:49,878 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-03-13 00:19:50,748 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 806
}

2022-03-13 00:19:50,850 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 14949
}

2022-03-13 00:19:50,953 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 14949
}

2022-03-13 00:19:51,069 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 14949
}

2022-03-13 00:19:51,171 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:51,274 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:51,376 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:51,479 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:51,581 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:51,684 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:52,549 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}

2022-03-13 00:19:53,201 INFO    MainThread:17669 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14949
  total_bytes: 14949
}
local_info {
}

2022-03-13 00:19:54,456 INFO    MainThread:17669 [wandb_run.py:_append_files():2180] logging synced files
